regolith/main.py

```diff
@@ -257,6 +257,20 @@ def create_parser():
     return p
 
 
+def merge_nested_dicts(dict0: dict, dict1: dict):
+    """Merge dict1 into dict0 recursively."""
+    for key in dict1.keys():  # iterate keys in dict1
```

Comment:
please remove comments, the intent should be interpretable from the code itself.

Reply:
Comments are removed.